README.md

```diff
@@ -26,6 +26,10 @@ create your local Config `~/.pse_timecard.json`
 
 `password` must be hidden with `base64`
 
+```bash
+$ echo -n "my-password" | bash64
```

Comment:
do you mean `echo -n "your password" | base64` ?

Reply:
yes, just fixed it